src/frontend/scss/colors/_theme.scss

```
@@ -233,6 +236,14 @@ $r-theme: (
     text-hover-color: r-color('grey59'),
     base-border: r-color('azure2'),
   ),
+  dashboard-credit-cards: (
+    empty-color: r-color(slate-grey),
+  ),
+  dashboard-credit-card-box: (
+    base-color: r-color(charcoal),
+    title-color: r-color(slate-grey),
+    expire-soon-color: r-color('orange'),
```

What do you think to use an already existing color for the demo site ?
```suggestion
    expire-soon-color: r-color('indianred3'),
```